tests/health_mgmtapi/health-cmdapi-test.sh.in

```diff
@@ -1,58 +1,78 @@
 #!/usr/bin/env bash
```

Comment:
- The changes you made were on an older version of this file. A PR has changed it in the meantime. So take what's on the master branch and then apply your changes on top of it.
- You have no idea what you're testing with the check_list function, since you didn't initialize that file yourself. For that test, the script would need to send a few commands to the running netdata, restart netdata and then validate the result, using the reusable function you'll see in the next comment.
- There are two tests here that should be executed for each of the other test cases (i.e. every time something changes): 
  - Does the LIST command show what it's supposed to show after each change? 
  - Is the saved JSON file the same?

Suggestion, so you can easily implement this: 
- use the API to first get the responses after each test step. Temporarily modify the test script so that it save each response after a call toa uniquely named file (e.g. testcase-$TESTID.json). 
- validate the files visually (i.e. they do look correct)
- add those files to github, so they'll always be there when the test suite runs.
- modify the previous saving logic and convert it to a check: Pass that number "TESTID" to a new, check_list function, which will have a similar logic to the check function. The test is: Issue LIST and save the output to `actual-$TESTID".json`. Then you do two diffs:   testcase-$TESTID.json vs actual-$TESTID".json and then testcase-$TESTID.json  vs what was written in varlibdir. All those files should be the same.  
- at the end, remove all the actual-* files from the test.

Reply:
I did not test after all the tests, because according with the newest algorithm this will only happen case we send in the query string requests that have more than "cmd=YYY", so I am using every time that the query string will change the server configuration, but case you continue finding necessary, I can change without any problem.
I used "echo " while I was developing and everything looked fine now, please confirm.

Reply:
This last commit is bringing the fix in the health cmd API that allow us to do all the tests correctly, at least I believe everything is ok. 
I also removed all the PYTHON dependency, the unique thing that I did not follow it was to remove the files at the end, I decided to do download of the file and compare with the file expected, case everything is ok, I am removing the file and moving in front, but case there is a problem I am keeping the file with the problem for the user to compare with the expected file.